game-core/src/main/java/games/strategy/engine/chat/Chat.java

```diff
@@ -34,15 +36,12 @@
   private final String chatChannelName;
   private final String chatName;
   private final SentMessagesHistory sentMessages;
-  private volatile long chatInitVersion = -1;
```

Comment:
Now that this is final, will the comment line down on L197 still be accurate?
```
 // will queue until we receive the init return value (they queue since they see the init version is -1)
```

Reply:
Good spot, it isn't.
I will have a look at that tomorrow.